src/components/PreActivityCard/index.js

```diff
@@ -27,128 +28,129 @@ const PreActivityCard = ({
-            <Text
+            {title}
+          </Text>
+          {timeEst ? (
+            <View
               style={{
-                color: '#808080',
-                fontSize: 17,
-                fontStyle: 'normal',
-                fontWeight: '500',
-                lineHeight: 20,
-                letterSpacing: 0,
-                textAlign: 'left'
+                flexDirection: 'row',
+                alignItems: 'center',
+                marginBottom: 24
               }}>
-              {' '}
-              {timeEst} min activity
-            </Text>
-          </View>
-        ) : (
-          <Space index={3} />
-        )}
-        <Text
-          style={{
-            fontSize: 17,
-            fontStyle: 'normal',
-            fontWeight: '500',
-            lineHeight: 24,
-            letterSpacing: 0,
-            textAlign: 'left'
-          }}>
-          {description}
-        </Text>
-      </View>
-      <View
-        style={{
-          width: '100%',
-          flexDirection: 'column',
-          alignItems: 'center'
-        }}>
-        <TouchableOpacity
-          onPress={handleBtn1}
-          style={{
-            borderRadius: 100,
-            width: '80%',
-            height: 52,
-            justifyContent: 'center',
-            backgroundColor: '#193340'
-          }}>
+              <FeatherIcons name="clock" size={20} color={'#808080'} />
+              <Text
+                style={{
+                  color: '#808080',
+                  fontSize: 17,
+                  fontStyle: 'normal',
+                  fontWeight: '500',
+                  lineHeight: 20,
+                  letterSpacing: 0,
+                  textAlign: 'left'
+                }}>
+                {' '}
+                {timeEst} min activity
+              </Text>
+            </View>
+          ) : (
+            <Space index={3} />
+          )}
           <Text
             style={{
               fontSize: 17,
+              fontStyle: 'normal',
               fontWeight: '500',
-              textAlign: 'center',
-              color: '#fff'
+              lineHeight: 24,
+              letterSpacing: 0,
+              textAlign: 'left'
             }}>
-            Get Started
+            {description}
           </Text>
-        </TouchableOpacity>
-        <TouchableOpacity
-          onPress={handleBtn2}
+        </View>
+        <View
           style={{
-            borderRadius: 100,
-            justifyContent: 'center',
-            marginTop: 20
+            width: '100%',
+            flexDirection: 'column',
+            alignItems: 'center'
           }}>
-          <Text
+          <TouchableOpacity
+            onPress={handleBtn1}
             style={{
-              fontSize: 17,
-              fontWeight: '500',
-              textAlign: 'center',
-              color: '#808080'
+              borderRadius: 100,
+              width: '80%',
```

Comment:
use { width: width - (left and right margin) }

Reply:
Look at use { width: width - (left and right margin) }

If you look at ReflectionHome, it has some examples for responsiveness